app/models/manageiq/providers/vmware/infra_manager/inventory/persister.rb

```diff
@@ -40,6 +40,9 @@ def initialize_inventory_collections
     add_collection(infra, :vm_parent_blue_folders)
     add_collection(infra, :vm_resource_pools)
     add_collection(infra, :root_folder_relationship)
+    add_collection(infra, :orchestration_templates) do |builder|
```

Comment:
For this to work we need to add the `has_many :orchestration_templates` association to the vmware infra_manager

Reply:
Yeah, I have it in manageiq repo.

Reply:
:+1: that works too, we'll just need to add that assoc to the base InfraManager class then too

Reply:
I thought about having it here. But the CloudManager had the corresponding one in manageiq.